tony-history-server/app/hadoop/Configuration.java

```diff
@@ -20,20 +20,25 @@
   private static final String CORE_SITE_CONF = YarnConfiguration.CORE_SITE_CONFIGURATION_FILE;
   private static final String HDFS_SITE_CONF = "hdfs-site.xml";
 
-  private static HdfsConfiguration conf = new HdfsConfiguration();
+  private static HdfsConfiguration hdfsConf = new HdfsConfiguration();
+  private static YarnConfiguration yarnConf = new YarnConfiguration();
```

Comment:
Can we just use a single `Configuration` that we use for both HDFS and YARN stuff? If we switch to `Configuration`, I think we might need to add `yarn-site.xml` to the resources loaded (YarnConfiguration probably looks for and loads `yarn-site.xml` automatically, but I don't think `Configuration` does).